app/src/main/webapp/WEB-INF/jsp/tiles/views/download.jsp

```diff
@@ -0,0 +1,54 @@
+<%@ page pageEncoding="UTF-8" contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<div class="row">
+    <div class="12 columns">
+        <h2>Download</h2>
+
+        <h3>Via FTP</h3>
+        <p>
+            You can download data for every dataset individually in Expression Atlas through our
+            <a href="http://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/experiments/" target="_blank">FTP site</a>.
```

Comment:
This is linking to bulk experiments, please point at the right URL.

Reply:
Doesn't this url contains both single cell and bulk experiments? For example, I can see [E-MTAB-5061](http://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/experiments/E-MTAB-5061/) there which is a single cell experiment.

Reply:
Could you please raise this issue with curators and data production? I could be wrong and we want both sites to point at a centralised directory with both bulk and SC experiments.

Reply:
sure will ask curators/data prod team!